slash/core/result.py

```diff
@@ -246,6 +250,13 @@ def add_skip(self, reason, append=True):
             self._skips.append(reason)
         context.reporter.report_test_skip_added(context.test, reason)
 
+    def get_execution_time(self):
```

Comment:
I think a better term for this would be `duration`. Also, consider making this a `timedelta` object to avoid confusion - or change the method name to `get_duration_seconds` (I personally prefer the former)